### DISCUSSION TITLE
Feature request - omit download directory

### DISCUSSION THREAD

**Discussion by sturia** [slskd/slskd#945] [Ideas]
In soulseek client there's an option to avoid recreating locally the directory containing the files that are being downloaded and download all the files in _Download directory_ directly.
Is it possible to have such a possibility in slskd as well? this way navigating through downloads is easier. 
thanks

**Comment by jpdillingham** [comment]
It's in the backlog, captured in #401.  It's lower on the list of priorities but I can see a few people are asking about it so I might see if there's a quick solution.
